README.md

```diff
@@ -58,7 +58,7 @@ docker-compose up -d postgis
 docker-compose run --rm \
   -e osmupdate_extra_params="--base-url=download.geofabrik.de/europe/switzerland-updates/" \
   -e osm_planet_mirror="http://download.geofabrik.de/" \
-  -e osm_planet_path_relative_to_mirror="europe/switzerland-160321.osm.pbf" \
+  -e osm_planet_path_relative_to_mirror="europe/switzerland-160301.osm.pbf" \
```

Comment:
Because geofabrik keeps the files from the first of month around longer than the rest.